mail/views.py

```diff
@@ -52,3 +54,34 @@ def post(self, request, *args, **kargs):  # pylint: disable=unused-argument, no-
                 } for batch_num, resp in enumerate(mailgun_responses)
             }
         )
+
+
+class FinancialAidMailView(APIView):
+    """
+    View for sending financial aid emails to individual learners
+    """
+    authentication_classes = (authentication.SessionAuthentication, )
+    permission_classes = (permissions.IsAuthenticated, UserCanMessageLearnersPermission, UserCanEditFinancialAid)
+
+    def post(self, request, *args, **kwargs):
+        """
+        Post request to send emails to an individual learner
+        """
+        try:
+            subject = request.data.get('email_subject') or kwargs['email_subject']
```

Comment:
No worries!  Thanks for clearing things up.  @noisecapella I've marked this as waiting for author, and I'll change it back once we've implemented this change.